docs/release-notes/NuGet-6.12.md

```diff
@@ -17,6 +15,139 @@ NuGet distribution vehicles:
 
 <sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
 
-## Not yet released
+## Summary: What's New in 6.11.1
+
+NuGet 6.12.1 is available in Visual Studio 17.12.0 and the.NET 9.0.101 SDK.
+
+### Issues fixed in this release
+
+* Deserializing an empty version range in a package dependency fails in .NET SDK 9.0.100-rc.2 [#13869](https://github.com/NuGet/Home/issues/13869)
+
+## Summary: What's New in 6.12
+
+NuGet 6.12.0 is available in the .NET 9.0.100 SDK.
+
+* Add property for toggling the to the previous NuGet resolver: RestoreUseLegacyDependencyResolver - [#13700](https://github.com/NuGet/Home/issues/13700)
```

Comment:
If we have the new dependency resolver chnage near the top here, do we still need this?